Junk4Dollars-appUITests/Junk4Dollars_appUITests.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Junk4Dollars_appUITests.swift
+//  Junk4Dollars-appUITests
+//
+//  Created by John Riccardi on 10/28/19.
+//  Copyright © 2019 John Riccardi. All rights reserved.
+//
```

Comment:
Mind changing the settings to remove this automatic comment?

Reply:
It looks like this involves copying / altering templates from the Xcode app itself. Do you think it's worth it? https://stackoverflow.com/questions/33720/change-templates-in-xcode/33743#33743